packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,240 @@
+| label              | label of input                                                                                                       | string                                                          |                   |
+| accessibilityLabel | aria-label of input when label is not defined                                                                        | string                                                          |                   |
+| rows               | height restrictions of input ([Checkout "Inactive - Active States" Usage](#with-different-inactive---active-states)) | '1' \| '3' \| 'expandable'                                      | '3'               |
+| showTags           | Boolean to show or hide tags inside AutoComplete ([Checkout "With Tags Outside" Usage](#with-tags-outside))          | boolean                                                         | true              |
+| value              | Controlled state of the value inside input                                                                           | string                                                          |                   |
+| onChange           | Callback when input value changes                                                                                    | (inputValue: string) => void                                    |                   |
+| selected           | Controlled state of the selected items ([Checkout "Controlled AutoComplete" Usage](#controlled-autocomplete))        | string[]                                                        |                   |
+| onSelectChange     | Callback when selected items change                                                                                  | ({ name, values }: { name: string; values: string[] } ) => void |                   |
+
+## Usage
+
+AutoComplete will be built as a new trigger for existing Dropdown component.
+
+Depending on where and how AutoComplete is used, the API can differ. Checkout examples below on how code will look like for a particular usage.
+
+- [Basic Usage](#basic-usage)
+- [Controlled AutoComplete](#controlled-autocomplete)
+- [In BottomSheet](#in-bottomsheet)
+- [With Tags Outside](#with-tags-outside)
+- [With Different Inactive - Active States](#with-different-inactive---active-statess)
+
+### Basic Usage
+
+```jsx
+<Dropdown selectionType="multiple">
+  <AutoComplete label="Select Cities" />
+  <DropdownOverlay>
+    <ActionList>
+      <ActionListItem title="Mumbai" value="mumbai" />
+      <ActionListItem title="Pune" value="pune" />
+      <ActionListItem title="Hyderabad" value="hyderabad" />
+    </ActionList>
+  </DropdownOverlay>
+</Dropdown>
+```
+
+### Controlled AutoComplete
+
+There are 3 controlled states that are relevant to AutoComplete
+
+1. Controlled state of filtering items
+2. Controlled state of the Input where user types
+3. Controlled state of selected items
+
+```jsx
+<AutoComplete
```

Comment:
Didn't understand 🤔 You mean move that full code example from inside of `<details>` to here? the `onChange` is required because that makes the AutoComplete filter its items.

Reply:
no don't move the full example but a few imp parts. right now it's hard to understand how/where is this `cities` coming from so filtering criteria is looking very confusing

you can just use the state variable, single-line and inputvaluechange and abstract out rest of the parts